packages/web/src/scss/components/FileUploader/_theme.scss

```
@@ -1,27 +1,32 @@
-@use '@tokens' as tokens;
+@use '@global' as global-tokens;
+@use '../../settings/globals';
 
-$row-gap: tokens.$space-600;
+$row-gap: global-tokens.$space-700;
```

I'd rename this to `$list-row-gap` or `$list-gap` because it's used only there.

```suggestion
$list-row-gap: global-tokens.$space-700;
```

Then you can move it between input and attachment variables.